CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes for each version of this project will be documented in this
     - Setting `rowSelectable` to `true` enables multiple row selection for the `igx-grid` component. Adds a checkbox column that allows (de)selection of one, multiple or all (via header checkbox) rows.
     - For more information about the `rowSelectable` property and working with grid row, please read the `igxGrid`'s [ReadMe](https://github.com/IgniteUI/igniteui-angular/blob/master/src/grid/README.md) about selection or see the [official documentation](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid-selection.html)
 - Added `onContextMenu` output to `igxGrid` to emit the clicked cell.
+- Added `onDoubleClick` output to `igxGrid` to emit the double clicked cell.
```

Comment:
This should be moved under 6.0.0

Reply:
@Aleksandyr create a new section called 6.0.0 and move it there

Reply:
@kdinev @zdrawku [PR](https://github.com/IgniteUI/igniteui-angular/pull/1300) was created.